modules/docs/README.md

```diff
@@ -0,0 +1,95 @@
+| is_valid_event                   | check if the event is valid according to the stream connector params                                                                                         | [Documentation](sc_event.md#is_valid_event-method)                   |
+| is_valid_neb_event               | check if the neb event is valid according to the stream connector params                                                                                     | [Documentation](sc_event.md#is_valid_neb_event-method)               |
+| is_valid_host_status_event       | check the "host status" event is valid according to the stream connector params                                                                              | [Documentation](sc_event.md#is_valid_host_status_event-method)       |
+| is_valid_service_status_event    | check the "servce status" event is valid according to the stream connector params                                                                            | [Documentation](sc_event.md#is_valid_service_status_event-method)    |
+| is_valid_host                    | check if the host name and/or ID are valid according to the stream connector params                                                                          | [Documentation](sc_event.md#is_valid_host-method)                    |
+| is_valid_service                 | check if the service description and/or ID are are valid according to the stream connector params                                                            | [Documentation](sc_event.md#is_valid_service-method)                 |
+| is_valid_event_states            | check if the state (HARD/SOFT), acknowledgement state and downtime state are valid according to the stream connector params                                  | [Documentation](sc_event.md#is_valid_event_states-method)            |
+| is_valid_event_status            | check if the status (OK, DOWN...) of the event is valid according to the stream connector params                                                             | [Documentation](sc_event.md#is_valid_event_status-method)            |
+| is_valid_event_state_type        | check if the state (HARD/SOFT) of the event is valid according to the stream connector params                                                                | [Documentation](sc_event.md#is_valid_event_state_type-method)        |
+| is_valid_event_acknowledge_state | check if the acknowledgement state of the event is valid according to the stream connector params                                                            | [Documentation](sc_event.md#is_valid_event_acknowledge_state-method) |
+| is_valid_event_downtime_state    | check if the downtime state of the event is valid according to the stream connector params                                                                   | [Documentation](sc_event.md#is_valid_event_downtime_state-method)    |
```

Comment:
So I guess that we have the choice to use is_valid_event_states or to use this 3 methods ?